src/theme.js

```diff
@@ -47,10 +47,24 @@ const theme = createTheme({
 			paper: "#FFFFFF",
 			default: "#FFFFFF",
 		},
+		input: {
```

Comment:
Uite aici de exemplu am adaugat o sectiune de culori `custom` - in cazul cand avem ceva culori extra care nu se regasesc in paleta principala.